server/server.go

```diff
@@ -0,0 +1,123 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+
+	"github.com/Clever/cron-admin/db"
+)
+
+func byteHandler(handler func(*http.Request) ([]byte, error)) func(http.ResponseWriter, *http.Request) {
+	return func(rw http.ResponseWriter, req *http.Request) {
+		data, err := handler(req)
+		if err != nil {
+			fmt.Printf("Received handler err: %s\n", err)
```

Comment:
hmm the lack of error handling here makes me question the `byteHandler`/`jsonHandler` pattern here. There's no way to determine if this should be a 4XX or 5XX

looks like this pattern originated in hall-monitor via @jefff so maybe he has some thoughts